lib/clearance/configuration.rb

```diff
@@ -133,6 +133,16 @@ def user_id_parameter
     def routes_enabled?
       @routes
     end
+
+    # Reloads the clearance user model class.
+    # This is called from the Clearance engine to reload the configured
+    # user class during each request while in development mode.
+    # @private
+    def reload_user_model
+      if @user_model.present?
+        @user_model = @user_model.to_s.constantize
```

Comment:
Yes, we basically have to force Ruby to re-look-up the constant, which is what we accomplish with `constantize` from a string. In an actual application running in development mode, if you make changes to the user class, the reference saved by clearance won't actually be the same class that you get with this constantize dance.

 As an added benefit (perhaps) this also is more permissive in configuration as users could pass a string or symbol which would still be constantized properly.